lib/ice_cube/validated_rule.rb

```diff
@@ -51,6 +42,14 @@ def other_interval_validations
       Array(@validations[base_interval_validation.type])
     end
 
+    def other_fixed_value_validations
+      @validations.values.flatten.select { |v|
```

Comment:
nit: `do / end` on multi-line blocks

Reply:
Ok, sure. 😄 

I've always been more partial to [Jim Weirich's semantic brace style][1], and I try to sneak it in when I can:

- Use { } for blocks that return values (functional)
- Use do / end for blocks that are executed for side effects (procedural)

It actually turns out that doing it this way, it works out that _very often_ it's the same as the single-line vs. multi-line style, because of the kinds of things that go into procedural vs. functional blocks.

[1]: http://archive.is/96ko8

Reply:
Okay I can be down with that - I like the idea :)